compiled_starters/haskell/your_program.sh

```
@@ -21,4 +21,4 @@ set -e # Exit early if any commands fail
 #
 # - Edit this to change how your program runs locally
 # - Edit .codecrafters/run.sh to change how your program runs remotely
-exec $(stack path --local-install-root)/bin/hs-grep-clone-exe "$@"
+exec $(stack path --local-install-root)/bin/codecrafters-grep-exe "$@"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)